<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RogaMonsterEditor</class>
 <widget class="QMainWindow" name="RogaMonsterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RogaMonsterEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="monsterListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="addMonsterButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Shared/ui.qrc">
            <normaloff>:/resource/add.png</normaloff>:/resource/add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeMonsterButton">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Shared/ui.qrc">
            <normaloff>:/resource/remove.png</normaloff>:/resource/remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="IDEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Spawn Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="spawnAnimationEdit">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Death Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="deathAnimationEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="faceIconBox">
        <property name="title">
         <string>FaceIcon</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="verticalGroupBox_2">
            <property name="title">
             <string>Monster Graphic</string>
            </property>
            <layout class="QVBoxLayout" name="monsterGraphicBox"/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="monsterGraphicSelectButton">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="verticalGroupBox">
          <property name="title">
           <string>SkillList</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>14</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>SkillID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Count</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="addMonsterDataButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../Shared/ui.qrc">
                 <normaloff>:/resource/add.png</normaloff>:/resource/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeMonsterDataButton">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../Shared/ui.qrc">
                 <normaloff>:/resource/remove.png</normaloff>:/resource/remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gridGroupBox">
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Attack</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="waitSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Wait</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="lifeSpinBox">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Defense</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Life</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="attackSpinBox">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="defenseSpinBox">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gridGroupBox_2">
        <property name="title">
         <string>Used Chain Effect</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Undead</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="undeadCombobox"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Human</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="humanCombobox"/>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Rizard</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rizardCombobox"/>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Beast</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="beastCombobox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Ground</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="groundCombobox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Wind</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="windCombobox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Fly</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="flyCombobox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Fire</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fireCombobox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Water</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="waterCombobox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Earth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="earthCombobox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Slash</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="slashCombobox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Pierce</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pierceCombobox"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Blunt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="bluntCombobox"/>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Plant</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="plantCombobox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset resource="../Shared/ui.qrc">
          <normaloff>:/resource/play.png</normaloff>:/resource/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave"/>
    <addaction name="actionSet_Resource_Dir"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionDamage_Editor">
   <property name="text">
    <string>Damage Editor</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSet_Resource_Dir">
   <property name="text">
    <string>Set Resource Dir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Shared/ui.qrc"/>
 </resources>
 <connections/>
</ui>
